spec/unit/pdk/util_spec.rb

```diff
@@ -349,7 +349,7 @@
       end
 
       it 'returns a path inside the system opt directory' do
-        is_expected.to eq(File.join('opt', 'puppetlabs', 'pdk', 'config'))
+        is_expected.to match(%r{\A/opt/})
```

Comment:
We can make this more specific if we want, but this now test now better matches it's description.